### DISCUSSION TITLE
Development Update - Migrate from SECRET_KEY to NAUTOBOT_SECRET_KEY

### DISCUSSION THREAD

**Discussion by jvanderaa** [nautobot/nautobot-app-chatops#220] [General]
Looks like there was an update recently that requires a migration from `SECRET_KEY` to `NAUTOBOT_SECRET_KEY` in your development environment. You may see an error like this:

```
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```

**Comment by jvanderaa** [comment]
@snaselj are there any other changes that have happened to the development environment for those that had an environment previous to some updates?

**Reply by snaselj** [reply]
That was due to the alignment with the cookiecutter template v1.0.0.

Changes to the creds.example.env are here: https://github.com/nautobot/nautobot-plugin-chatops/commit/c03835e911c7ffdcd86f56af904c6313c71872d0#diff-701b8134d2cc200f1476ee00d98760fd5ed05356a080f1e89792ecd9eb388989

Changes to development.env are here (dev.env => development.env): https://github.com/nautobot/nautobot-plugin-chatops/commit/c03835e911c7ffdcd86f56af904c6313c71872d0#diff-0ad04d13ec6b61526eb1c55ca29d24056576577af05eab7c2231c4a7027e549d

Environment variables were prefixed with `NAUTOBOT_`.
